.travis.yml

```diff
@@ -52,6 +52,9 @@ jobs:
 
 script: ./scripts/runTestsOnTravis.sh $TEST_SUITE
 
+before_deploy:
+  - curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin
```

Comment:
I think we need to also fetch a recent `cosign` version since the default one that is installed will be picked up by the script might be too old to be compatible, then override `COSIGN_BINARY` to use that one when calling it

Reply:
Cosign is used for verifying the checksum of Syft itself and only used when the `-v` option is set when running the script. Ironically, it is not in the default one-liner (for which I copied from the website). However, I will add since it is extra validation.

Reply:
Thanks for clarifying 👍